### DISCUSSION TITLE
Using any Filament form component in Blade files

### DISCUSSION THREAD

**Discussion by df999** [filamentphp/filament#6328] [Help] [answered]
Is it possible to reuse any of the built-in Filament form fields in a Blade file for custom use?

Eg. let's say I have a Reports section in my app where I need to change the report date using AlpineJS.

My intuition is to use the Filament date time picker in a blade file to change the report date:

```html
<div x-data="{ currentDate: ''  }">
    <x-filament::date-time-picker x-model="currentDate" />
    <x-my-sales-report current-date="currentDate" />
</div>
```

Am I going about this completely wrong? What would be a good approach here?

**Comment by danharrin** [comment, marked as answer]
No there isn't a way to do that at the moment, our views are tightly integrated with our PHP classes

**Reply by df999** [reply]
Okay thanks Dan - great to know.

I was thinking maybe I could create a dynamic wrapper component like this:

`<livewire:field :component="DatePicker::make('current_date')" x-model="currentDate" />`

...but I guess having each field as a form may generate a lot of overhead...

**Reply by danharrin** [reply]
Yeah, use the form builder normally where you can that was what it was made for.
